jest.config.js

```diff
@@ -4,7 +4,8 @@ module.exports = {
   ...baseConfig,
   projects: [
     // Our front-end
-    '<rootDir>/src/web/app/jest.config.js',
+    // To do: Add front-end tests
```

Comment:
Can you add the URL to the issue https://github.com/Seneca-CDOT/telescope/issues/3655 here in this comment too?

Reply:
Yup. Added.